data_summaries/Medical Meadow.json

```diff
@@ -0,0 +1,455 @@
+        "Collection": "Medical Meadow",
+        "Collection URL": "https://github.com/kbressem/medAlpaca",
+        "Languages": [
+            "English"
+        ],
+        "Task Categories": [
+            "BioMedical Question Answering",
+            "Instructional Question Answering",
+            "Medical Question Answering",
+            "Question Answering"
+        ],
+        "Text Sources": [],
+        "Model Generated": [
+            "OpenAI GPT-3.5-turbo"
+        ],
+        "Format": [
+            "Zero-shot"
+        ],
+        "Human Annotation": "No",
+        "Derived from Datasets": [
+            "PubMed Causal Language Use"
+        ],
+        "Creators": [
+            "University Hospital Aachen",
+            "Technische Universität München",
+            "Berliner Hochschule für Technik"
+        ],
+        "Licenses": [
+            {
+                "License": "GNU General Public License v3.0",
+                "License URL": "https://github.com/junwang4/causal-language-use-in-science/blob/master/LICENSE"
+            }
+        ],
+        "License Notes": "The license was provided in the github repo of the original dataset",
+        "License Verified By": "",
+        "Dataset Filter IDs": [
+            "medical-meadow-health-advice"
+        ]
+    },
+    "medical-meadow-pubmed-causal": {
+        "Unique Dataset Identifier": "medical-meadow-pubmed-causal",
+        "Dataset Name": "pubmed-causal",
+        "Paper Title": "Detecting Causal Language Use in Science Findings",
+        "Dataset URL": "https://aclanthology.org/D19-1473/",
+        "GitHub URL": "https://github.com/junwang4/causal-language-use-in-science",
+        "Hugging Face URL": "https://huggingface.co/datasets/medalpaca/medical_meadow_pubmed_causal",
+        "Papers with Code URL": "",
+        "ArXiv URL": "",
+        "Semantic Scholar Corpus ID": 202787823,
+        "Collection": "Medical Meadow",
+        "Collection URL": "https://github.com/kbressem/medAlpaca",
+        "Languages": [
+            "English"
+        ],
+        "Task Categories": [
+            "BioMedical Question Answering",
+            "Instructional Question Answering",
+            "Medical Question Answering",
+            "Question Answering"
+        ],
+        "Text Sources": [],
+        "Model Generated": [
+            "OpenAI GPT-3.5-turbo"
+        ],
+        "Format": [
+            "Zero-shot"
+        ],
+        "Human Annotation": "No",
+        "Derived from Datasets": [
+            "PubMed Causal Language Use"
+        ],
+        "Creators": [
+            "University Hospital Aachen",
+            "Technische Universität München",
+            "Berliner Hochschule für Technik"
+        ],
+        "Licenses": [
+            {
+                "License": "GNU General Public License v3.0",
+                "License URL": "https://github.com/junwang4/causal-language-use-in-science/blob/master/LICENSE"
```

Comment:
If it doesn't say anywhere in the repo that this is the license for the data, then we treat this as a code license. Later we run code to extract the github licenses programmatically, in case people want to use that, but we try not to assume its the data license.

Could we instead put the license for PubMed here (as I understand from the paper, that is the only thing they use aside from OpenAI models). (Also add OpenAI terms here.)

Reply:
I updated it to "unspecified" license instead of the one from the github repo. What do you think?
@shayne-longpre @jadkabbara